app/src/main/java/org/mozilla/fenix/components/bookmarks/BookmarksUseCase.kt

```diff
@@ -41,7 +45,8 @@ class BookmarksUseCase(storage: BookmarksStorage) {
     }
 
     class RetrieveRecentBookmarksUseCase internal constructor(
-        private val storage: BookmarksStorage
+        private val bookmarksStorage: BookmarksStorage,
+        private val historyStorage: HistoryStorage
```

Comment:
Can `HistoryStorage` be optional?  This way if preview image is not required, there's no need to have access to `HistoryStorage`

Reply:
Sure, made it optional.